### ISSUE TITLE
Web portal issue

### ISSUE THREAD

**Issue by jeffreyty2008** [apioo/fusio#131]
Hi, I have installed and tried out the "apiVersion" 2.0.0 and 3.0.0 but i found that the web portal for both of them are still at 0.5.0 version. Half of the features in the 0.5.0 web portal is like not working. Is there a place that i can upgrade the web portal version?

**Comment by k42b3** [comment]
Hi, so the web portal (backend app) is basically a separate software with its own version number independent of the Fusio project. At the moment the backend app is at version 0.5. Could you tell me which parts of the app are not working and do you have maybe some error logs with more details about the error?

**Comment by jeffreyty2008** [comment]
Here is one of the problems that I have encountered.
So I was trying to create a schema through the backend app. Every time I filled out "name" and the "JsonSchema", I clicked save and nothing happened.
The same thing happens to creating users in the backend app.
Is there any way to create a schema in fusio except using the backend app?

**Comment by k42b3** [comment]
Yes, if you have access to the CLI you can use the system command i.e.:

```
php bin/fusio schema:add test-schema foo.json
```

This creates a new schema with the name "test-schema". The file `foo.json` contains a json schema.
